content/en/docs/components/misc/nuclio.md

```diff
@@ -190,7 +190,7 @@ def nuc_pipeline(
 ```
 the code above assumes nuclio was deployed into the `nuclio` namespace on the same cluster, when using a remote cluster or a different namespace you just need to change the `dashboard` URL.
 
-See [nuclio pipline components](https://github.com/kubeflow/pipelines/tree/master/components/nuclio) (allowing to deploy, delete, or invoke functions) 
+See [nuclio pipeline components](https://github.com/kubeflow/pipelines/tree/master/components/nuclio) (allowing to deploy, delete, or invoke functions) 
```

Comment:
Thanks @kelvins !

Can you please also take a look at line 26?

```diff
- Nuclio functions can be used in the following ML pipline tasks:
+ Nuclio functions can be used in the following ML pipeline tasks:
```

Reply:
@Bobgy This guide maye not be up-to-date for KF v1.1 (check out the banner). Do you know who can test this, since @jlewi and @sarahmaddox were the original guide authors, I think. Let me know if you need me to step in and test this.

Reply:
@kelvins 

Lines 41-42: Can you also help fix these sentences here? 

- Form a singular verb by adding `s` to `contains` (as in: "the repo... contains")
- `Git repo` -> `GitHub repo`
- `. can also follow...` (there's a second person pronoun missing at the beginning of this sentence) -> `. You can also follow...`
- Add a reference to `O'Reilly Katacoda` that hosts the tutorial.

```diff
- Nuclio [Git repo](https://github.com/nuclio/nuclio) contain detailed documentation on the installation and usage.
- can also follow this [interactive tutorial](https://www.katacoda.com/javajon/courses/kubernetes-serverless/nuclio).
+ The Nuclio [GitHub repo](https://github.com/nuclio/nuclio) contains detailed documentation on the installation and usage.
+ You can also follow this [interactive Nuclio tutorial](https://www.katacoda.com/javajon/courses/kubernetes-serverless/nuclio) by O'Reilly Katacoda.
```

Reply:
Line 75:
- Add a comma after an introductory word (`First, ...`)
- Replace the pronoun: in general, the docs should use second person (`you`) instead of first person (`we`)
- Add the definite article `the` -> `the ... package`
- Add an apostrophe in `won't`

```diff
- First we need to import `nuclio` package (we add an `ignore` comment so this line wont be compiled later):
+ First, you need to import the `nuclio` package (note that you add an `ignore` comment, so that this line won't be compiled later):
```

Reply:
Lines 126, 180: spelling inside command and Python code blocks:

```diff
- event = nuclio.Event(body=b'good morninng')
+ event = nuclio.Event(body=b'good morning')
```

```diff
def nuc_pipeline(
-   txt='good morningf',
+   txt='good morning',
):
```

Reply:
Update: Line 193:

- Add a full stop at the end of the sentence
- For a11y, maybe we can swap `See` with `Refer to`

```diff
- See [nuclio pipeline components](https://github.com/kubeflow/pipelines/tree/master/components/nuclio) (allowing to deploy, delete, or invoke functions)
+ Refer to [nuclio pipeline components](https://github.com/kubeflow/pipelines/tree/master/components/nuclio) (allowing to deploy, delete, or invoke functions).
```

Reply:
Done :heavy_check_mark: